subsys/fs/nvs/nvs.c

```diff
@@ -629,7 +629,16 @@ static int nvs_gc(struct nvs_fs *fs)
 			continue;
 		}
 
+#ifdef CONFIG_NVS_LOOKUP_CACHE
+		wlk_addr = fs->lookup_cache[nvs_lookup_cache_pos(gc_ate.id)];
+
+		if (wlk_addr == NVS_LOOKUP_CACHE_NO_ADDR) {
```

Comment:
This cannot happen unless something is wrong with the cache, instead of jumping over I would rather fallback on setting it to fs->ate_wra.

Reply:
I agree with you